### PR TITLE
feat(RHTAPREL-615): add rh-push-to-registry-redhat-io Pipeline

### PR DESCRIPTION
**Pull request by happybhati**
This Pipeline will be used to push content to registry.redhat.io.

### CODE REVIEW

**Review by scoheb — APPROVED** [review]

File: pipelines/rh-push-to-registry-redhat-io/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# rh-push-to-registry-redhat-io pipeline
+
+Tekton pipeline to push images to a registry.redhat.io registry.
```
Comment (kasemAlem) [inline pipelines/rh-push-to-registry-redhat-io/README.md:3]:
```suggestion
Tekton pipeline to push images to the registry registry.redhat.io .
```

File: pipelines/rh-push-to-registry-redhat-io/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| release | The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution | No | - |
+| releaseplan | The namespaced name (namespace/name) of the releasePlan | No | - |
+| releaseplanadmission | The namespaced name (namespace/name) of the releasePlanAdmission | No | - |
+| releasestrategy | The namespaced name (namespace/name) of the releaseStrategy | No | - |
```
Comment (kasemAlem) [inline pipelines/rh-push-to-registry-redhat-io/README.md:12]: According to [PR#273](https://github.com/redhat-appstudio/release-service/pull/273) once it's merged then RleaseStrategy won't be supported anymore, isn't better we prepare this pipeline to meet that PR ?

**Review by mmalina — COMMENTED** [review]
I have some comments... But I guess too late. Maybe they can be addressed in a follow up PR?

**Comment by happybhati** [comment]
@mmalina and @kasemAlem, Thanks for pointing out recent changes. I will fix those in the follow-up PR.

File: pipelines/rh-push-to-registry-redhat-io/README.md (hunk 2 above)
Comment (johnbieren) [reply pipelines/rh-push-to-registry-redhat-io/README.md:12]: Yes, +1

**Comment by happybhati** [comment]
@johnbieren, @mmalina , @kasemAlem I have addressed above issue in #228 Thanks for review and sorry for inconvenience.

> I'd guess it is in the data.json but we should discuss it


I will discuss `tagPrefix` with Leandro and add later.

File: pipelines/rh-push-to-registry-redhat-io/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+| enterpriseContractPolicy | JSON representation of the policy to be applied when validating the enterprise contract | No | - |
+| enterpriseContractPublicKey | Public key to use for validation by the enterprise contract | Yes | k8s://openshift-pipelines/public-key |
+| extraConfigGitUrl |URL to the remote Git repository containing the extra config | No | - |
+| extraConfigGitRevision | Revision to fetch from the remote Git repository containing the extra config | No | - |
+| extraConfigPath | Path to the extra config file within the repository | No | - |
+| tag | The default tag to use when mapping file does not contain a tag | No | - |
+| addGitShaTag | When pushing the snapshot components, also push a tag with the image git sha | Yes | true |
+| addSourceShaTag | When pushing the snapshot components, also push a tag with the image source sha | Yes | true |
+| addTimestampTag | When pushing the snapshot components, also push a tag with the current timestamp | Yes | false |
+| pyxisServerType | The Pyxis server type to use. Options are 'production' and 'stage' | No | - |
```
[the remark this answers is not in the document]
Comment (kasemAlem) [reply pipelines/rh-push-to-registry-redhat-io/README.md:23]: +1

**Pull request merged**